djstripe/models.py

```diff
@@ -441,26 +446,28 @@ def sync_charges(self, cu=None, **kwargs):
 
     def sync_current_subscription(self, cu=None):
         cu = cu or self.stripe_customer
-        sub = cu.subscription
+        sub = getattr(cu, 'subscription', None)
```

Comment:
What do you think about changing "sub" to "stripe_subscription" and "cur_sub" to "current_subscription"?